components/brave_shields/core/common/DEPS

```diff
@@ -1,5 +1,6 @@
 include_rules = [
   "+components/content_settings",
   "+components/grit/brave_components_strings.h",
+  "+components/prefs",
```

Comment:
@mkarolin this dep is not appropriate for core/common because PrefService can only be accessed from the browser process

Reply:
sorry, missed this. this dep will be removed in https://github.com/brave/brave-browser/issues/50414